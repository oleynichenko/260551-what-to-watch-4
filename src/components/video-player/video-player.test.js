import VideoPlayer from "@components/video-player/video-player";

const movie = {
  id: 5,
  title: `Revenant`,
  image: `img/revenant.jpg`,
  preview: `path`
};

describe(`VideoPlayerComponent`, () => {
  it(`should render VideoPlayer`, () => {
    const tree = renderer.create(
        <VideoPlayer
          isPlaying={false}
          poster={movie.image}
          src={movie.preview}
          isMuted={true}
        />,
        {
          createNodeMock: () => {
            return {};
          }
        }
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
