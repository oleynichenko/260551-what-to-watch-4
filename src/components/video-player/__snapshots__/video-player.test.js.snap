// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayerComponent should render VideoPlayer 1`] = `
<video
  className="player__video"
/>
`;
